src/NuGet.Indexing/NuGetQuery.cs

```diff
@@ -194,6 +197,11 @@ private static void AuthorClause(BooleanQuery query, Analyzer analyzer, IEnumera
             query.Add(ConstructClauseQuery(analyzer, "Authors", values), occur);
         }
 
+        private static void PackageTypeClause(BooleanQuery query, Analyzer analyzer, IEnumerable<string> values, Occur occur)
```

Comment:
In terms of query performance, you may want to make the package types behave like the owners (make it a filtered query).

If it's a clause, Lucene will have more documents to verify. If it's a filtered query, result set is made very small at the start of the query and Lucene will only have a few documents to check.